pkg/collect/host_sysctl.go

```diff
@@ -0,0 +1,45 @@
+package collect
+
+import (
+	"bytes"
+	"os/exec"
+
+	"github.com/pkg/errors"
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+)
+
+var _ HostCollector = (*CollectHostSysctl)(nil)
+
+var execCommand = exec.Command
+
+const HostSysctlPath = `host-collectors/system/sysctl.txt`
+
+type CollectHostSysctl struct {
+	hostCollector *troubleshootv1beta2.HostSysctl
+	BundlePath    string
+}
+
+func (c *CollectHostSysctl) Title() string {
+	return hostCollectorTitleOrDefault(c.hostCollector.HostCollectorMeta, "Sysctl")
+}
+
+func (c *CollectHostSysctl) IsExcluded() (bool, error) {
+	return isExcluded(c.hostCollector.Exclude)
+}
+
+func (c *CollectHostSysctl) Collect(progressChan chan<- interface{}) (map[string][]byte, error) {
+
+	cmd := execCommand("sysctl", "-a")
```

Comment:
Good point, let me 👀

Reply:
Looks like this could be a good candidate - https://github.com/lorenzosaino/go-sysctl - implementation wise is relatively simple and seems to be well maintained.